### DISCUSSION TITLE
Filter disappears randomly when selecting various vertical tabs

### DISCUSSION THREAD

**Discussion by Jrstew2016** [microsoft-search/pnp-modern-search#3349] [Q&A]
I have 1 filter web part, 1 vertical web part and 6 Search results web parts. The filter is aligned to open when any one of the verticals is selected. Each vertical tab is set to only display the results in the results web part when it's selected.
When I click on the first Vertical tab (Everyone) the filter displays. 2nd vertical tab, it might display. Click on say the next 4 tabs and the filter is gone. If I go back and select the first tab again, the filter displays. So then if I go click on any vertical, it'll appear. Once I click on another tab that's not the Everyone tab, the filter is gone again. 
In Edit mode, they work fine. No issues at all (meaning it doesn't give the error that because something isn't selected it won't show results). So I know everything is configured right.

I provided a video clip [here](https://drive.google.com/file/d/1zi1o7a8sA3tNycw9naI0FccPT0j0yQKc/view?usp=sharing) that shows the behavior. Is this a bug? Or do I need to have a filter for each Search results?

I'm using v4.9.1 with a custom display template based off of the People card. Other than that, nothing else is custom.

**Comment by whitehotaru** [comment]
Are there any errors displayed in the browser console? You can reach it through the dev tools: https://developer.chrome.com/docs/devtools/open/

<img width="424" alt="image" src="https://github.com/microsoft-search/pnp-modern-search/assets/18051300/fcbe5d2a-f62b-478a-a89e-beb6c634a5fc">

**Reply by Jrstew2016** [reply]
Have not tried that. I'll check it out though and keep you posted. Thanks!

**Reply by Jrstew2016** [reply]
Here's all the error's I'm getting. I'm not a developer so most of it is greek to me. But there seems to be a lot of errors with the word 'filter_panel' in it. Not sure if that means anything in regards to the Filters web part. But I've include all screenshots of the 5 errors and un-collapsed. Only 1 error had 2 different messages so I didn't screenshot all 22, 17 and so on of the errors on some because it's the same message. Hopefully that makes sense.
I might also add. You'll see an error with the image and text being needed. Sometimes the users images show up. And say 6 hours later (maybe after a crawl or something), they're just a place holder. I believe this is what's being used to display it. <img class="image_profile" src={{item.PictureURL}}  width="92" height="92" ></img>

If adding the template code might help, let me know and I'll provide that. Maybe it'll give a clue to whatever's going on with the Filter and image issue.

![2023-11-22 23_16_27-_Untitled - Notepad](https://github.com/microsoft-search/pnp-modern-search/assets/149169590/56252038-89d5-4cae-b232-6ed58d5e1ae7)
![2023-11-22 23_17_08-_Untitled - Notepad](https://github.com/microsoft-search/pnp-modern-search/assets/149169590/26a35496-619a-4c58-a233-35be15f31d09)
![2023-11-22 23_17_28-_Untitled - Notepad](https://github.com/microsoft-search/pnp-modern-search/assets/149169590/f9ce911a-f71b-4393-9a35-b5b0a27f787b)
![2023-11-22 23_17_57-_Untitled - Notepad](https://github.com/microsoft-search/pnp-modern-search/assets/149169590/484d8122-f213-4181-9872-67fe24f0f0ac)
![2023-11-22 23_19_00-_Untitled - Notepad](https://github.com/microsoft-search/pnp-modern-search/assets/149169590/6920506c-12f3-488c-8e52-0689bb575f37)
